code/game/objects/structures/crates_lockers/closets/l3closet.dm

```diff
@@ -12,7 +12,7 @@
 
 /obj/structure/closet/l3closet/general/WillContain()
 	return list(
-		/obj/item/clothing/suit/storage/hooded/ds_bio_suit,
```

Comment:
I just made this a month or two ago, using new sprites from cre77

Reply:
Implementation was of the item was really bad. Cre also doesn't like them and wants to redo them. For now, they are replaced on the human side by the new sprite.